qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/jupyterhub/main.tf

```diff
@@ -30,7 +41,16 @@ resource "helm_release" "jupyterhub" {
         shared-pvc        = var.shared-pvc
         conda-store-pvc   = var.conda-store-pvc
         conda-store-mount = var.conda-store-mount
-        extra-mounts      = var.extra-mounts
+        extra-mounts      = merge(
```

Comment:
I think this is `concat` no? Merged will combine the dicts https://www.terraform.io/language/functions/merge.

Reply:
![image](https://user-images.githubusercontent.com/51954708/156613849-cb44dc3c-376e-49b1-943f-5b891a320b60.png)

Reply:
as `var.extra-mounts` is already a dict of the form 
```
var.extra-mounts = {
   "/dask-etc" = {}
}
```
and extra-mounts expect a dict for `jsonencode`, so we will be merging those dicts indeed, but only the new key `idle-culler-...` will be included, the values are maintained the same.